### PR TITLE
Add darkmode variables and toggle

### PR DESCRIPTION
**Pull request by lop-Louis**
- added storybook addons for dark mode toggle
- added some variables for poc on darkmode discussion with sebastian
- added decorators to apply dark mode in stories not only outer frame
- add global types in storybook toolbar to switch between themes
- replace existing color stories with color palatte in figma

#### house keeping
- grouping default variables on top, and override on the bottoms

### CODE REVIEW

**Review by frankforland — COMMENTED** [review]
Looks good but this does not address the branding issue right?

**Review by frankforland — COMMENTED** [review]
Lest discuss on daily

File: src/stories/Colors/Colors.module.css [hunk 1]
```diff
@@ -1,30 +1,44 @@
...
+  padding-left: 0.5rem;
+}
+.title {
+  margin-bottom: 0.5rem;
+  margin-left: 0.5rem;
+}
+.colorPalette {
+  display: flex;
+  flex-direction: column;
+  gap: var(--grid-gutter);
```
Comment (bwg29nov) [inline src/stories/Colors/Colors.module.css:12]: Removed `grid-gutter` , add if you required

**Review by ulrichgreen — APPROVED** [review]
Ship it!

**Pull request merged**